### ISSUE TITLE
[1.16.5] Torcherino not working on some blocks

### ISSUE THREAD

**Issue by Peca21** [NinjaPhenix/Torcherino#18]
Torcherino doesn't work on some blocks. (Example blocks: thermal blocks like furnace and mekanism miner) but works on others like vanilla furnace and Industrial foregoing blocks. I thought some mods disabled tick acceleration but acceleration wand from NotEnoughWands works fine on every block.
![Uploading 2021-08-08_20.56.38.png…]()

**Comment by Antraxs** [comment]
Yeah it seems to be a hit or miss with most mods, hopefully we get an update soon.

**Comment by niklas2233** [comment]
Would like to add that it doesn't work on BotanyPots while time in a bottle works
